src/System.Management.Automation/resources/RemotingErrorIdStrings.resx

```diff
@@ -1636,7 +1636,10 @@ All WinRM sessions connected to Windows PowerShell session configurations, such
   <data name="InvalidRoleCapabilityFileExtension" xml:space="preserve">
     <value>The provided role capability file {0} does not have the required .psrc extension.</value>
   </data>
-  <data name="SSHAbruptlyTerminated" >
+  <data name="SSHAbruptlyTerminated" xml:space="preserve">
     <value>The SSH transport process has abruptly terminated causing this remote session to break.</value>
   </data>
+  <data name="WSManClientDllNotAvailable" xml:space="preserve">
+    <value>WSMan client library is not installed</value>
```

Comment:
Agreed.  It sounds like this only affects Nano.  So maybe something like "WSMan client library is not installed.  This parameter set requires WSMan for PowerShell remoting commands such as Invoke-Command, New-PSSession, Enter-PSSession."  The problem is that SSH remoting may be available if SSH client is installed.  But I don't know if that is a likely scenario.